src/main/kotlin/com/exactpro/th2/lwdataprovider/entities/requests/SseMessageSearchRequest.kt

```diff
@@ -39,7 +39,10 @@ data class SseMessageSearchRequest(
     val attachedEvents: Boolean,
     val lookupLimitDays: Int?,
     val resumeFromIdsList: List<StoredMessageId>?,
-    val onlyRaw: Boolean
+    @Deprecated("Use responseFormats instead", ReplaceWith("responseFormats",
+        "com.exactpro.th2.lwdataprovider.entities.requests"), DeprecationLevel.WARNING)
```

Comment:
We cannot do that replacement because types does not match. Please, leave only a message